restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/mybatis/MybatisIncomingPhoneNumbersDao.java

```diff
@@ -326,7 +234,7 @@ private IncomingPhoneNumber toIncomingPhoneNumber(final Map<String, Object> map)
         map.put("date_updated", DaoUtils.writeDateTime(incomingPhoneNumber.getDateUpdated()));
         map.put("friendly_name", incomingPhoneNumber.getFriendlyName());
         map.put("account_sid", DaoUtils.writeSid(incomingPhoneNumber.getAccountSid()));
-        map.put("phone_number", incomingPhoneNumber.getPhoneNumber());
+        map.put(PHONE_NUM, incomingPhoneNumber.getPhoneNumber());
```

Comment:
@jaimecasero why don't we move all string to constant strings?

Reply:
i just moved the ones i modifed, but we could potentially move all...